resources/deployment.yaml

```diff
@@ -47,10 +47,10 @@ spec:
       - name: objectstorage-provisioner-sidecar
         image: quay.io/containerobjectstorage/objectstorage-sidecar:latest
         args:
-          - "--connect-address=$(CONNECT_ADDRESS)"
-        env:
-        - name: CONNECT_ADDRESS
+          - "--driver-addr=$(DRIVER_ADDRESS)"
+        envFrom:
+        - name: DRIVER_ADDRESS
           valueFrom:
             secretKeyRef:
               name: objectstorage-provisioner
-              key: LISTEN_ADDRESS
+              key: DRIVER_ADDRESS
```

Comment:
Fails to create the deployment because the correct format for this is:
```
        env:
        - name: DRIVER_ADDRESS
          valueFrom:
            secretKeyRef:
              name: objectstorage-provisioner
              key: DRIVER_ADDRESS
```

Reply:
driver changes are WIP